### PR TITLE
Implement script for US 304

### PR DESCRIPTION
**Pull request by Minh-Tester**
US 304-Auto_User is able to change pw
https://dev.azure.com/RookiesBatchVI/RookiesBatch6Group4/_sprints/taskboard/Rookies%20Batch%206%20-%20Group%204/RookiesBatch6Group4/Sprint%202%20-%20Finish%20manage%20asset,%20management%20assignment%20and%20manage%20request%20for%20returning%20assets?workitem=304

image

### CODE REVIEW

File: AssetManagementTestProject/PageObj/ChangePasswordPage.cs [hunk 1]
```diff
@@ -12,21 +12,22 @@ public class ChangePasswordPage : WebDriverAction
     private string btnSaveNewPw = "//button[contains(@type, 'submit')]";
     private string btnChangePw = "//a[contains(@href, '/change-password')]";
     private string btnCancel = "//span[contains(text(), 'Cancel')]";
-    private string headerChangePw = "//h1[text()='Change Password']";
+    public string HeaderChangePw = "//h1[text()='Change Password']";
     private string pathChangePw = "change-password";
-    private string textChangePwSuccessfully = "//p[text()='Your password has been changed successfully']";
+    public string TextChangePwSuccessfully = "//p[text()='Your password has been changed successfully']";
```
Comment (honganh1206) [inline AssetManagementTestProject/PageObj/ChangePasswordPage.cs:17]: Make accessbility of all properties in each page object either all public or all private + Follow naming conventions for public/private variables

File: AssetManagementTestProject/TestSetup/Constant.cs [hunk 2]
```diff
@@ -15,8 +14,12 @@ public class Constant
     #region CREATED USERNAMES-PWS
     public const string STAFF_USERNAME = "anhp4";
     public const string STAFF_PASSWORD = "Staff@1234";
+    public const string CHANGED_USER_PASSWORD = "Staff@12345";
```
Comment (honganh1206) [inline AssetManagementTestProject/TestSetup/Constant.cs]: These variables should have their own files in the Test Data folder e.g., LoginTestData.cs

File: AssetManagementTestProject/PageObj/LoginPage.cs [hunk 3]
```diff
@@ -3,8 +3,8 @@
 namespace AssetManagementTestProject.PageObj;
 public class LoginPage : WebDriverAction
 {
-    private readonly string tfUsername = "//input[contains(@id, 'username')]";
-    private readonly string tfPassword = "//input[contains(@id, 'password')]";
+    public readonly string TfUsername = "//input[contains(@id, 'username')]";
+    public readonly string TfPassword = "//input[contains(@id, 'password')]";
     private readonly string btnLogin = "//button[contains(@type, 'submit')]";
```
Comment (honganh1206) [inline AssetManagementTestProject/PageObj/LoginPage.cs]: Change these properties as my comment in the ChangePasswordPage

File: AssetManagementTestProject/PageObj/ChangePasswordPage.cs (hunk 1 above)
Comment (Minh-Tester) [reply AssetManagementTestProject/PageObj/ChangePasswordPage.cs:17]: Done

File: AssetManagementTestProject/PageObj/LoginPage.cs (hunk 3 above)
Comment (Minh-Tester) [reply AssetManagementTestProject/PageObj/LoginPage.cs]: Done

**Pull request merged**